core/backend/package.json

```diff
@@ -98,7 +98,7 @@
     "webpack": "^5.76.0"
   },
   "dependencies": {
-    "@bentley/imodeljs-native": "4.6.6",
+    "@bentley/imodeljs-native": "4.6.3",
```

Comment:
@DanRod1999 why dont you merge the native changes you want into this pr and abandon yours?